mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/release.xml

```diff
@@ -1 +1 @@
+<release xmlns="http://musicbrainz.org/ns/mmd-2.0#" id="871f12ec-6259-4e93-b440-9108559c38b1"><title>Chiptunes = WIN \m|♥|m/</title><status>Official</status><text-representation><language>eng</language><script>Latn</script></text-representation><artist-credit><name-credit><artist id="2b542c4a-694a-49b9-b475-9f55f99436a1"><name>Chiptunes = WIN \m|♥|m/</name><sort-name>Chiptunes = WIN \m|♥|m/</sort-name><alias-list><alias sort-name="Chiptunes = WIN">Chiptunes = WIN</alias></alias-list></artist></name-credit></artist-credit><release-group id="268d91a0-1a7f-4600-bfd4-74d738eb7de4" type="Compilation"><primary-type>Album</primary-type><secondary-type-list><secondary-type>Compilation</secondary-type></secondary-type-list></release-group><date>2012-06-05</date><country>XW</country><release-event-list><release-event><date>2012-06-05</date><area id="525d4e18-3d00-31b9-a58b-a146a916de8f"><name>[Worldwide]</name><sort-name>[Worldwide]</sort-name><iso-3166-1-code-list><iso-3166-1-code>XW</iso-3166-1-code></iso-3166-1-code-list></area></release-event></release-event-list><barcode/><label-info-list><label-info><label id="7669cf9a-7328-44cf-96ba-b2dc9d534ed8"><name>Chiptunes = WIN \m|♥|m/</name></label></label-info></label-info-list><medium-list count="1"><track-count>51</track-count><medium><format>Digital Media</format><disc-list count="0"/><track-list count="51"/></medium></medium-list><tag-list><tag count="1"><name>bitunes</name></tag><tag count="1"><name>bandcamp</name></tag><tag count="2"><name>chiptunes</name></tag></tag-list></release>
```

Comment:
There seems to a regression with the `packaging` field.
*Production Behavior*
```json
packaging: "None",
packaging-id: "119eba76-b343-3e02-a292-f0f00644bb9b",
``` 

*Current Behavior*
```json
packaging: {
 "name": "None",
  "id": "119eba76-b343-3e02-a292-f0f00644bb9b"
}
```

Reply:
Not sure what you call “production” and “current” behaviors?

* [Search for this release](http://musicbrainz.org/ws/2/release/?query=reid:871f12ec-6259-4e93-b440-9108559c38b1&fmt=json) (currently in production) returns:

  ```json
  packaging: {
    "name": "None",
    "id": "119eba76-b343-3e02-a292-f0f00644bb9b"
  }
  ```

* [Lookup at this release](http://musicbrainz.org/ws/2/release/871f12ec-6259-4e93-b440-9108559c38b1?fmt=json) (currently in production) returns:

  ```json
  packaging: "None",
  packaging-id: "119eba76-b343-3e02-a292-f0f00644bb9b",
  ```

This is a known issue: SEARCH-579.

Reply:
Yes, This is is what I was referring to. To be sure, we want that the Lookup behavior should be followed by the search as well.

Reply:
Yes.